app/javascript/components/admin/AdminTable.js

```diff
@@ -521,18 +523,29 @@ class AdminTable extends React.Component {
 
     const handleSuccess = response => {
       if (response && response.data && response.data.user_rows) {
+        const keysToSkip = ['active_state', 'lock_reason', 'auto_lock_reason'];
         // NOTE: react-csv has a bug where false values don't show up in the downloaded CSV
         // This has been addressed and recently merged: https://github.com/react-csv/react-csv/pull/193
         // Once this gets released and we update our version this code won't be necessary.
-        const csvData = response.data.user_rows.map(userData => {
-          return {
-            ...userData,
-            notes: removeFormulaStart(userData.notes).toString(),
-            is_api_enabled: userData.is_api_enabled.toString(),
-            is_2fa_enabled: userData.is_2fa_enabled.toString(),
-            is_locked: userData.is_locked.toString(),
-          };
+        const keysWithBoolValues = ['is_api_enabled', 'is_2fa_enabled', 'is_locked'];
+
+        let csvData = [];
+        response.data.user_rows.forEach(row => {
+          let csvObj = {};
```

Comment:
you could use `_.omit` here instead of the loop to remove the `keysToSkip` and the bring back the simplicity of the `csvData` being generated by the .map

Reply:
I definitely agree with restoring the simplicity of `getCSVData`. 

`_.omit` is being removed from lodash in v5.0.0 due to performance issues: [https://github.com/lodash/lodash/issues/2930](https://github.com/lodash/lodash/issues/2930). 

I've made an update that uses destructuring to omit the keys as one other option. Let me know what you think of this approach. One downside is use of `// eslint-disable-next-line no-unused-vars`

See [c28fbc8a54fbdf6f25351ef078d8e3cdcf2c6f7](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/fc28fbc8a54fbdf6f25351ef078d8e3cdcf2c6f7)

Reply:
This is good to know about omit - I'm pretty sure it is used elsewhere